visual-dotnet/SauceLabs.Visual/Models/SelectiveRegion.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using OpenQA.Selenium;
+using SauceLabs.Visual.GraphQL;
+using SauceLabs.Visual.Utils;
+
+namespace SauceLabs.Visual.Models
+{
+    public class SelectiveRegion
+    {
+        public IgnoreRegion? Region { get; set; }
+        public IWebElement? Element { get; set; }
+
+        public DiffingOption[]? EnableOnly { get; set; }
+        public DiffingOption[]? Disable { get; set; }
```

Comment:
In JS we have called it `enableOnly` and `disableOnly`. Shall do the same here?

Reply:
Of course we do, I was nearly sur I read `enableOnly` / `disable`, will address that.